kafka_producer.js

```diff
@@ -216,4 +235,5 @@ app.get('/dbp/:uuid',providerCallbackHandlers.delhiPlasmaBankHandler);
 
 app.get('/tickethandler/freshdesk/wfcreate', providerCallbackHandlers.freshdeskTicketWfCreationHandler);
 
-app.get('/messagehandler/:providerid/:apikey', providerCallbackHandlers.providerMessageHandler);
\ No newline at end of file
+app.get('/messagehandler/:providerid/:apikey', providerCallbackHandlers.providerMessageHandler);
+app.post('/rawdata/:apikey', providerCallbackHandlers.rawdata);
```

Comment:
add a path param :providerId so that same api can be used for multiple providers

`/rawdata/:providerId/:apikey`